site/en/workflows.md

```diff
@@ -942,13 +1023,39 @@ In this processing step, the text output of the OCR or post-correction can be ev
   </tbody>
 </table>
 
-## Format Conversion (Optional)
+## Generic Data Management (Optional)
 
 OCR-D produces PAGE XML files which contain the recognized text as well as detailed
 information on the structure of the processed pages, the coordinates of the recognized
-elements etc. Optionally, the PAGE XML can be converted to a different output format.
+elements etc. Optionally, the output can be converted, changed or dummy output can be generated. 
+
+### Step 18: Adaptation of Coordinates
+Some processors change the original image by e.g. scaling, deskewing or cropping it and produce alternative images, whose coordinates differ
+from the original images. Nevertheless, all output from the OCR-D-Processors sticks to the coordinates of the original images. If you want to 
+visualize the output with the produced alternative images, the coordinates have to be re-calculated accordingly.
```

Comment:
Not sure if that's really worth an extra section – it could well be subsumed under _format conversion_

```suggestion
### Step 18: Adaptation of Coordinates
All OCR-D processors are required to relate coordinates to the original image for each page, and to keep the original image reference (`Page/@imageFilename`). However, sometimes it may be necessary to deviate from that strict requirement in order to get the overall workflow to work. 

For example, if you have a page-level dewarping step, it is currently impossible to correctly relate to the original image's coordinates for any segments annotated after that, because there is no descriptive annotation of the underlying coordinate transform in PAGE-XML. Therefore, it is better to _replace the original image_ of the output PAGE-XML by the dewarped image before proceeding with the workflow. If the dewarped image has also been cropped or deskewed, then of course all existing coordinates are re-calculated accordingly as well.

Another use case is exporting PAGE-XML for tools that cannot apply cropping or deskewing, like [LAREX](https://github.com/OCR4all/LAREX) or Transkribus.
```

Reply:
Not sure whether this statement is correct for Transkribus BTW